Resources/Locale/en-US/store/categories.ftl

```diff
@@ -1,16 +1,16 @@
-﻿# Uplink
+# Uplink
 store-category-debug = debug category
 store-category-debug2 = debug category 2
-store-category-weapons = Weapons
+store-category-weaponry = Weaponry
```

Comment:
Is the breaking change to the locid worth it.

Reply:
Probably not; wasn't aware it'd be a breaking change. Will revert it to store-category-weapons.